openlattice.yaml

```diff
@@ -4816,6 +5291,91 @@ components:
           type: integer
         count:
           type: integer
+    CollectionTemplateType:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: An optional UUID for the collection template type.
+        name:
+          type: string
+          description: The unique name of the collection template type.
+        title:
+          type: string
+          description: The friendly name for the collection template type.
+        description:
+          type: string
+          description: A description of the collection template type.
+        entityTypeId:
+          type: string
+          format: uuid
+          description: The id of the entity type that this collection template type wraps.
+    EntityTypeCollection:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+          description: An optional UUID for the entity type collection.
+        type:
+          $ref: "#/components/schemas/fullQualifiedName"
```

Comment:
If we are capitalizing thing, can we pls capitalize all components?

Reply:
I'd have to look into the downstream effects of our example notebooks and code, so I created a jira-ticket for this: https://jira.openlattice.com/browse/LATTICE-2396
The reason it was done for EntitySet, was because it wasn't consistent inside the document.